viewer/viewer.js

```diff
@@ -971,9 +971,7 @@ function expireDevice (nodes, dirs, minFreeSpaceG, nextCb) {
     }
 
     if (data.hits.total <= 10) {
```

Comment:
I think that this check is wrong now, it should be:
```
if (data.hits.total == 0) {
```

Reply:
I checked again and everything works fine, sorry for the noise.

Reply:
np - hits.total is the number of matching documents ignoring pagination (from)